docs/Updating-gene-and-gene_alias-tables.md

```diff
@@ -3,14 +3,16 @@ This manual is intended for users that have knowledge about the structure of the
 
 When loading studies into cBioPortal it is possible for warnings to occur that are caused by an outdated seed database. Gene symbols can be deprecated or be assigned to a different Entrez Gene in a new release. Also Entrez Gene IDs can be added. This markdown explains how to update the seed database, in order to use the most recent Entrez Gene IDs.
 
-The cBioPortal scripts package provides a method to update the `gene` and `gene_alias` tables. This requires the latest version of the NCBI Gene Info.
+The cBioPortal scripts package provides a method to update the `gene` and `gene_alias` tables.
+
+## Prepare
 
 ### Human genes
-Homo_sapien.gene_info.gz\
-ftp://ftp.ncbi.nih.gov/gene/DATA/GENE_INFO/Mammalia/Homo_sapiens.gene_info.gz
+Download [gene_info.txt](https://github.com/cBioPortal/datahub-study-curation-tools/blob/master/gene-table-update/build-input-for-importer/gene_info.txt)  
```

Comment:
This fixed/constant file reference to the master branch of cBioPortal is a bit of a problem. The master branch may move forward and the gene_info.txt file may get updated, but how will the user notice that they are out of date and need to do a gene update? There is no versioning in the filename. I guess we could have a policy that users will download and import this file periodically whether there has been a change or not ... but that seems a little opaque. I think it would be good if we somehow captured what version of the gene table the users are currently using, and what version they are updating to.

Reply:
@yichaoS if there is versioning for the gene_info file, the documentation should explain how the versioning will be visible to users ... how users will be able to know if they are current, or if they need to do an update.

Reply:
@sheridancbio the most up-to-date input gene file for importer will always be here: https://github.com/cBioPortal/datahub-study-curation-tools/blob/master/gene-table-update/build-input-for-importer, named as `gene_info.txt`; the file name would not include the versioning (the original output would and it would be in the archive folder, we just copy the original and rename it as `gene_info.txt`).

Reply:
okay, so the user can keep track of their last installed version and either do a "diff" to the master branch, or they can look for when new entries appear in the archive folder. Thanks.